paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -510,7 +513,9 @@ internal class CustomerSheetViewModel @Inject constructor(
                     clientSecret = stripeIntent?.clientSecret,
                     shippingDetails = null,
                     draftPaymentSelection = null,
-                    onMandateTextChanged = { _, _ -> },
+                    onMandateTextChanged = { mandate, _ ->
```

Comment:
Should we add support for the showAbove here? Seems like it won't be used now, but seems like a place of bugs for future LPM support.